packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -571,21 +586,31 @@ contract DeployImplementationsInterop is DeployImplementations {
     function createOPSMContract(
         DeployImplementationsInput _dii,
         DeployImplementationsOutput,
-        OPStackManager.Blueprints memory blueprints
+        OPStackManager.Blueprints memory blueprints,
+        string memory release,
+        OPStackManager.ImplementationSetter[] memory setters
     )
         internal
+        virtual
```

Comment:
nit, it's harmless to have the virtual modifier here, but for clarity (since we don't expect people to override this in `DeployImplementationsInterop`) let's remove it
```suggestion
```

Reply:
Removed in: https://github.com/ethereum-optimism/optimism/pull/11875/commits/3e8c9f9661d80e4eeae14df9a6a53b7abba600c3